site/packages/aa-core/provider/sendUserOperation.md

```diff
@@ -65,7 +65,9 @@ A Promise containing the hash of the user operation and the request that was sen
 
 ## Parameters
 
-### `UserOperationCallData | UserOperationCallData[]`
+### `UserOperationCallData | UserOperationCallData[] | Hex`
+
+`UserOperationCallData` is an object with the following properties:
```

Comment:
```suggestion
`UserOperationCallData` is either a `Hex`value, or an object with the following properties:
```

Reply:
Ah, I didn't want to confuse that here actually... I add the `| Hex` explicitly already above to make that clearer. because otherwise it might be the case that you can do `Hex[]` as batch (which is not the case)